<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileOperation</class>
 <widget class="QWidget" name="fileOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="ip"/>
       </item>
       <item>
        <widget class="QLineEdit" name="port"/>
       </item>
       <item>
        <widget class="QPushButton" name="connectServer">
         <property name="text">
          <string>Config</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileNameInput"/>
       </item>
       <item>
        <widget class="QPushButton" name="selectFileBtn">
         <property name="text">
          <string>Select file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendBtn">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
